doc/tutorials/run_all_tests_with_docker.md

```diff
@@ -116,17 +119,21 @@ Just run this command:
 make install
 ```
 
-The number 10 can be changed as follows: number of supported simultaneous threads by your CPU + 2. But don't worry, this can only affect the speed of the building, it cannot really break it.
+After Elektra has been installed we need to add it to the PATH variable, meaning you and the tests can interact with Elektra by typing/executing `kdb` in the command line.
+
+```sh
+export PATH="$PWD/install/bin:$PATH"
+```
 
-### 4. Run tests
+### 4. Run Tests
 
 Finally run the tests. There are two sets of tests. Run the first one with this command:
 
 ```sh
 make run_all
 ```
 
-For the second set to run remember to execute `make install` from the previous step. Run the second set with this command:
+For the second set to run, remember to execute `make install` from the previous step. Run the second set with this command:
```

Comment:
To make `kdb run_all` work more than setting the PATH is needed. (Also `export LD_LIBRARY_PATH="$PWD/install/lib:$PATH"`).

But even then the testkdb_ensure fails.

Reply:
> What did you try? Can you write an issue about that please?

It was 4 months ago. I really do not remember what I did back then.

> Yes, I also wonder why you do not use docker directly?

My whole dev environment is inside this VM. On my host OS, Windows, it is hard to map volumes.  And there sometimes problems with the rights.

> Please report these three issues separately, they seems to be unrelated to this tutorial.

Ok, you mean
1. Failing tests
2. Windows installation problems
3. What is the 3rd issue?

Reply:
> To make `kdb run_all` work more than setting the PATH is needed. (Also `export LD_LIBRARY_PATH="$PWD/install/lib:$PATH"`).
> 
> But even then the testkdb_ensure fails.

Thank you! I will add it.

Reply:
As 3rd issue, you probably meant failing tests with kdb run_all. I added it as well.

Reply:
I fixed everything you requested. Is it OK now? I am really running extremely out of time with all these changes... 
The building seems to get stuck. Can you restart it?